frontend/services/viewer/viewer.tsx

```diff
@@ -908,9 +909,29 @@ export class ViewerService {
 		return this.showAll();
 	}
 
-	public async setCamera({ position, up, view_dir, look_at, animate, rollerCoasterMode, account, model }) {
+	public async setProjectionMode(mode) {
 		await this.isModelReady();
-		UnityUtil.setViewpoint(position, up, view_dir, look_at, animate !== undefined ? animate : true, rollerCoasterMode);
+		switch (mode) {
+			case VIEWER_PROJECTION_MODES.ORTHOGONAL:
+				UnityUtil.enableOrthographic();
+				break;
+			default:
+				UnityUtil.disableOrthographic();
+		}
+	}
+
+	public async setCamera({ position, up, view_dir, look_at, type, orthographicSize, account, model }) {
+		await this.isModelReady();
+		UnityUtil.setViewpoint(
+			position,
+			up,
+			view_dir,
+			look_at,
+			type === VIEWER_PROJECTION_MODES.ORTHOGONAL,
```

Comment:
just pass type down

Reply:
@carmenfan A lot of older viewpoints don't seem to have `type` defined, so if `type` is passed down here, a default projection type would have to be determined either in Unity Utils or Unity.

Reply:
if type is empty assume it's perspective. We can either code that logic inside unity.

Either way it needs to be in unity because it needs to work for embedded users without extensively playing with the schema